week-1/H-numbers/exercise.js

```diff
@@ -1 +1,5 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = 15;
+var numberOfMentors = 8;
+total = numberOfStudents + numberOfMentors;
```

Comment:
It compiles and runs fine, but remember about initialising your variables correctly - what are you missing on line 4?
Also, rethink the name ;)

Reply:
I missed to declare variable
variable name could be totalNumber
change done